project/plugins.sbt

```diff
@@ -1,3 +1,9 @@
-addSbtPlugin("com.jsuereth" % "xsbt-gpg-plugin" % "0.6")
+//resolvers += Resolver.url("sbt-plugin-releases", new URL("http://scalasbt.artifactoryonline.com/scalasbt/sbt-plugin-releases/"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("me.lessis" % "ls-sbt" % "0.1.2")
+//addSbtPlugin("com.jsuereth" % "xsbt-gpg-plugin" % "0.6")
+
+addSbtPlugin("me.lessis" % "ls-sbt" % "0.1.3")
+
+resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots"
+
+addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "3.0.0")
```

Comment:
ide support is really up to a developer. I don't use eclipse and now my plugin depends on an eclipse plugin. can this be removed?